model/database/spans.yaml

```diff
@@ -361,8 +368,7 @@ groups:
           For batch operations, if the individual operations are known to have the same operation name
           then that operation name SHOULD be used prepended by `BATCH `,
           otherwise `db.operation.name` SHOULD be `BATCH`.
-        requirement_level:
-          conditionally_required: If readily available.
+        requirement_level: required
```

Comment:
HBASE always has operation name and does not have query language, no reason for conditionally_required